### PR TITLE
chore(deps): remove unused dependencies

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves []

### Description

Removed:
- [x] @react-native-community/slider
- [x] @react-native-community/viewpager

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- remove unused dependencies - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by dimatretyak — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -106,8 +106,6 @@
     "@react-native-community/geolocation": "2.0.2",
     "@react-native-community/masked-view": "0.1.11",
     "@react-native-community/netinfo": "6.0.1",
-    "@react-native-community/slider": "4.2.1",
```
Comment (pvinis) [inline package.json:109]: i hate to block this, but this is needed by storybook. its not a direct dep unfortunately, so we have to have it ourselves 🤦

**Review by pvinis — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -106,8 +106,6 @@
...
-    "@react-native-community/viewpager": "^4.2.2",
```
Comment (pvinis) [inline package.json:110]: and this one is needed by `react-native-scrollable-tab-view`, in a similar way :(
https://github.com/artsy/eigen/pull/4362#discussion_r566846184

**Comment by pvinis** [comment]
I hate to close this, because it's such a great initiative, but unfortunately we can't/shouldn't remove these. I made https://github.com/artsy/eigen/pull/7017 instead, so keep some comments about the deps similar to these.

**Comment by gkartalis** [comment]
😅😅😅 will close it then

**Pull request closed without merging**